### DISCUSSION TITLE
Modify the scale of atlas

### DISCUSSION THREAD

**Discussion by Minako-375** [JingyiGF/HERBS#62] [Q&A]
I use juvenile Wistar rats and I want to use HERBS to visualize NP1.0 trajectory from multiple slices.
Since only Waxholm Space Rat atlas of adult rats is available, I wonder if I can modify the scale of atlas to adjust to juvenile rats. If I can, it is really helpful to know the location of each channel in the brain.
Also, is it possible to install Wistar rats atlas?

**Comment by JingyiGF** [comment]
Hi, if Wistar rat altas has a volumn version, you can upload the atlas to HERBS and do the tricks. Please see the tutorial in GitHub page https://github.com/JingyiGF/HERBS/blob/main/Tutorial/upload_and_process_user_defined_atlases.md

Let me know if you need more help :-)

**Reply by Minako-375** [reply]
Thank you.
Is it possible to modify the default Waxholm Space Rat atlas scale similar to that tutorial?

**Reply by JingyiGF** [reply]
well, you can change the resolution to scale the original atlas, but i do not think it is a correct thing to do for your research.

**Reply by Minako-375** [reply]
The best way is to get the juvenile Wistar rat atlas, but it might be difficult. So, an alternative idea is to adjust the brain size, ignoring the differences between SD and Wistar. 
From the brain histological slice, I can distinguish the size of the brain. Therefore, I wonder if I can modify the size of the brain of atlas so that the location of each channel is adjust to the brain that I use in my experiments.
